src/main/osd/osd_elements.c

```diff
@@ -1017,6 +1017,26 @@ static void osdElementPower(osdElementParms_t *element)
     tfp_sprintf(element->buff, "%4dW", getAmperage() * getBatteryVoltage() / 10000);
 }
 
+static void osdElementRcChannels(osdElementParms_t *element)
+{
+    const uint8_t xpos = element->elemPosX;
+    const uint8_t ypos = element->elemPosY;
+
+    for (int i = 0; i < OSD_RCCHANNELS_COUNT; i++) {
+        if (osdConfig()->rcChannels[i] >= 0) {
+            // Translate (1000, 2000) to (-1000, 1000)
+            int data = constrain((rcData[osdConfig()->rcChannels[i]] - 1500) * 2, -1000, 1000);
```

Comment:
Using `scaleRange(rcData[osdConfig()->rcChannels[i]], PWM_RANGE_MIN, PWM_RANGE_MAX, -1000, 1000)` makes this a bit more intuitive to understand.

Reply:
I knew it is there, but couldn't recall the name lol

Reply:
Hmm... `scaleRange` does mult and div. I wonder if it worth the use here...

Reply:
I think it's worth it, considering the code will be easier to read.